src/WattleScript.Interpreter/Execution/VM/Processor/Processor_InstructionLoop.cs

```diff
@@ -705,6 +715,8 @@ private void ExecArgs(Instruction I)
 				}
 			}
 		}
+
+		private int tmpI = 0;
```

Comment:
Accidentally committed?

Reply:
fixed in [9d78d89](https://github.com/WattleScript/wattlescript/pull/36/commits/9d78d899a2c1edb6f9650297c230cdabdf2bcb7f)